store/blocks.jsonc

```diff
@@ -1,23 +1,92 @@
 {
```

Comment:
`.jsonc` extension?

Reply:
Yeah, json with comments. The builder accepts it for the blocks file 😁